packages/forklift-console-plugin/src/modules/Plans/views/list/components/VMsProgressCell.tsx

```diff
@@ -53,19 +55,37 @@ export const VMsProgressCell: React.FC<CellProps> = ({ data }) => {
   const progressVariant = getPlanProgressVariant(phase);
 
   return (
-    <Link to={`${planURL}/vms`}>
-      <div className="forklift-table__status-cell-progress">
-        <Progress
-          width={100}
-          label={t('{{success}} of {{total}} VMs migrated', counters)}
-          valueText={t('{{success}} of {{total}} VMs migrated', counters)}
-          value={counters?.total > 0 ? (100 * counters?.success) / counters?.total : 0}
-          title={t(phaseLabel)}
-          size={ProgressSize.sm}
-          measureLocation={ProgressMeasureLocation.top}
-          variant={progressVariant}
-        />
-      </div>
-    </Link>
+    <div className="forklift-table__status-cell-progress">
+      <Progress
+        width={100}
+        label={
+          <Link to={`${planURL}/vms`}>{t('{{success}} of {{total}} VMs migrated', counters)}</Link>
+        }
+        valueText={t('{{success}} of {{total}} VMs migrated', counters)}
+        value={counters?.total > 0 ? (100 * counters?.success) / counters?.total : 0}
+        title={
+          <Popover
+            headerContent={<div>{t('Status details')}</div>}
+            bodyContent={
+              <div>
+                {t('Total of {{success}} VMs succeeded', counters)}
+                <br />
+                {t('Total of {{error}} VMs failed', counters)}
+                <br />
+                {t('Total of {{canceled}} VMs canceled', counters)}
+                <br />
+              </div>
+            }
+          >
+            <Button variant="link" isInline>
```

Comment:
Nice, can we add some hint the button is a popover, like an help icon `<HelpIcon />` or a dashed underline `<DescriptionListTermHelpTextButton ` can you add such a hint here ?

Reply:
Sure. I kept the same style as was before for the hyperlinks in this plans list table (i.e. no icon and no inline styling of underline) but I agree it's nicer to distinguish it as a popover..

Reply:
There are 2 options:
![Screenshot from 2024-09-24 23-51-53](https://github.com/user-attachments/assets/840e9a48-dda5-40e3-9149-dab7711f7fa3)

![Screenshot from 2024-09-24 23-53-10](https://github.com/user-attachments/assets/f7ed6720-3764-4ada-bb18-2c6f003d0eaa)


I choose the 1st option to be aligned with the `Warning` status